src/Apps/Auction/__tests__/redirects.test.ts

```diff
@@ -25,23 +25,23 @@ describe("Auction/redirects", () => {
   }
 
   const mockResolver = (
-    data: RegisterQueryResponse
-  ): RegisterQueryResponse => ({
+    data: DeFragedRegisterQueryResponse
+  ): DeFragedRegisterQueryResponse => ({
     sale: data.sale,
     me: data.me,
   })
 
   it("does not redirect if a sale is found", async () => {
-    const { redirect } = await render(
+    const result = await render(
```

Comment:
Why the change?

Reply:
Artefact of debugging when we broke the relay found router.

Reply:
(I can fix though)